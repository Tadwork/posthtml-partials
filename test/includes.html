<include src="test/include/partial.html"></include>

<partial name="say" from="included partial"></partial>
