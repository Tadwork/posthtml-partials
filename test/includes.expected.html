



	<p>Hello from included partial!</p>

